### ISSUE TITLE
Inner functions cause errors

### ISSUE THREAD

**Issue by bovlb** [alexhagen/jdc#3]
JDC seems to be doing the wrong thing with inner functions.

```
class Foo:
    pass

```
```
%%add_to Foo
def method(self):
    def inner_function(x): 
        return 2*x
    return inner_function(3)
```

> IndentationError: expected an indented block

**Comment by alexhagen** [comment]
Should be fixed as of release ``0.0.8`` on ``pip``

**Comment by bovlb** [comment]
Wow.  You're fast!
JDC now works as expected.  Thanks!
